resources/com.sap.piper/pipeline/stageDefaults.yml

```
@@ -70,7 +70,12 @@ stages:
           - 'userTokenCredentialsId'
           - 'whitesource/userTokenCredentialsId'
           - 'whitesourceUserTokenCredentialsId'
-  Performance: {}
+  Performance:
+    stepConditions:
```

The stepCondition for the `multicloudDeploy` step is missing here.
```suggestion
    stepConditions:
      multicloudDeploy:
        configKeys:
          - 'cfTargets'
          - 'neoTargets'
```